src/sidebar/util/groups.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/**
+ * Combine groups from multiple api calls together to form a unique list of groups.
+ * Add a listGroup property to each group identifying it as a "mygroup" or a "featured" group.
+ *
+ * @param {myGroups[]} groups the user is a member of
+ * @param {featuredGroups[]} groups scoped to the particular uri, authority, and user
+ */
+const combineGroups = (myGroups, featuredGroups) => {
```

Comment:
The convention in our code is to use `function <name> () { ... }` for named top-level functions in a module. Using `const name = <arrow function>` would be a reasonable choice in a new codebase, I'm just recommending this for consistency.